docs/multibrokers.md

```diff
@@ -0,0 +1,201 @@
+				zap.Any("error", err))
+
+			rest.ERROR(w, err)
+			return
+		}
+
+		sidecarAddress := environment.GetBrokerSpecificSidecarAddr(channelBroker)
+		sidecarWritePort := environment.GetBrokerWritePort(channelBroker)
+
+		reqAddress := fmt.Sprintf("%s:%s/%s", sidecarAddress, sidecarWritePort, channel)
+
+		logger.Debug("encoding message to Avro schema")
+
+		encodedMsg, err := encodeToAvro(channel, r.Body)
+		if err != nil {
+			logger.Error("unable to encode message to Avro schema",
+				zap.String("channel", channel),
+				zap.Any("error", err))
+
+			rest.ERROR(w, err)
+			return
+		}
+
+		logger.Info("sending message to broker",
+			zap.String("broker", channelBroker),
+			zap.String("channel", channel))
+
+		resp, err := sendRequest(reqAddress, encodedMsg)
+		if err != nil {
+			logger.Error("unable to send request to "+channelBroker+" sidecar",
+				zap.Any("error", err))
+
+			rest.ERROR(w, err)
+			return
+		}
+
+		rest.JSON(w, resp.StatusCode, resp.Body)
+	}
+}
+```
+
+To receive information from these brokers the Load Balancer  serves a `readMessageHandler` which is posted to by the broker specific sidecars, when these receive a message from the broker itself. The read handler then parses from the request the *Channel* targeted by that operation, verifying if this *Channel* is in fact part of the *Node*'s *Boundaries*, if it is it then **deserializes** the message received, from *Avro* to *Json* and sends it to the *Node*'s client.
+
+`readMessageHandler`:
+
+```go
+func (s *Server) readMessageHandler() rest.Handler {
+	return func(w http.ResponseWriter, r *http.Request) {
+		logger.Info("handling message read")
+
+		channel := strings.TrimPrefix(r.URL.Path, "/")
+
+		if !environment.InputChannelList().Contains(channel) {
+			logger.Error("channel " + channel + " not found in input channel list")
+			insprError := ierrors.NewError().
+				BadRequest().
+				Message("channel '%s' not found", channel)
+
+			rest.ERROR(w, insprError.Build())
+			return
+		}
+
+		clientReadPort := os.Getenv("INSPR_SCCLIENT_READ_PORT")
+		if clientReadPort == "" {
+			insprError := ierrors.NewError().
+				NotFound().
+				Message("[ENV VAR] INSPR_SCCLIENT_READ_PORT not found")
+
+			rest.ERROR(w, insprError.Build())
+			return
+		}
+
+		logger.Debug("decoding message from Avro schema")
+
+		decodedMsg, err := decodeFromAvro(channel, r.Body)
+		if err != nil {
+			logger.Error("unable to decode message from Avro schema",
+				zap.String("channel", channel),
+				zap.Any("error", err))
+
+			rest.ERROR(w, err)
+			return
+		}
+
+		logger.Info("sending message to node through: ",
+			zap.String("channel", channel))
+
+		reqAddress := fmt.Sprintf("http://localhost:%v/%v", clientReadPort, channel)
+
+		resp, err := sendRequest(reqAddress, decodedMsg)
+		if err != nil {
+			logger.Error("unable to send request from lbsidecar to node",
+				zap.Any("error", err))
+			rest.ERROR(w, err)
+			return
+		}
+
+		rest.JSON(w, resp.StatusCode, resp.Body)
+	}
+}
+```
+
```

Comment:
I don't see putting this code here without a detailed description of each part of it as any useful